### PR TITLE
Clean-up of Python 3 PR

### PR DESCRIPTION
**Pull request by corranwebster**
This is a cleanup of #261.

### CODE REVIEW

**Comment by corranwebster** [comment]
Tests are passing, but not ready to merge yet: 2to3 was run on the examples and so there's some manual back-porting that's needed (mainly `from __future__ import print_function`, I think).

File: chaco/color_mapper.py [hunk 1]
```diff
@@ -405,7 +404,7 @@ def _map(self, X):
 
         """
 
-        if type(X) in [IntType, FloatType]:
+        if type(X) in [int, float]:
```
Comment (mdickinson) [inline chaco/color_mapper.py]: Orthogonal to the PR (so please don't change it here), but it bugs me that this isn't `if instance(X, (int, float, long))`. Is it worth opening an issue to remind us to fix these explicit type checks? Or should we just wait for it to cause problems in something real?

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,21 @@
 language: python
 sudo: false
 python:
-  - 2.6
   - 2.7_with_system_site_packages
+  - 3.4
```
Comment (mdickinson) [inline .travis.yml]: Remind me: are we explicitly unsupporting Python 3.3, or is it just inconvenient to include it in the Travis versions?

File: chaco/color_mapper.py (hunk 1 above)
Comment (mdickinson) [inline chaco/color_mapper.py]: Though the right check is probably just an `np.isscalar(X)`.

File: .travis.yml (hunk 2 above)
Comment (corranwebster) [inline .travis.yml]: Travis infrastructure.  There's nothing that should stop it working on Python 3.3 I think, although if it did pose any difficulty then I'd be happy to drop support for 3.3 at this point.

File: examples/tutorials/scipy2008/custom_tool.py [hunk 3]
```diff
@@ -12,14 +12,14 @@ class CustomTool(BaseTool):
     event_state = Enum("normal", "mousedown")
 
     def normal_mouse_move(self, event):
-        print "Screen:", event.x, event.y
+        print("Screen:", event.x, event.y)
```
Comment (mdickinson) [inline examples/tutorials/scipy2008/custom_tool.py]: Missing a `from __future__ import print_function` in this module?

File: examples/tutorials/scipy2008/custom_tool_click.py [hunk 4]
```diff
@@ -10,13 +10,13 @@
 class CustomTool(BaseTool):
 
     def normal_mouse_move(self, event):
-        print "Screen point:", event.x, event.y
+        print("Screen point:", event.x, event.y)
```
Comment (mdickinson) [inline examples/tutorials/scipy2008/custom_tool_click.py]: Missing `print_function` import.

Comment (corranwebster) [inline examples/tutorials/scipy2008/custom_tool_click.py]: Yep - thank you, I thought I'd caught all of those.

**Comment by mdickinson** [comment]
Changes LGTM. One question about 3.3, and two places where there's a `print_function` import missing (unless you want to print tuples in Python 2).

File: chaco/color_mapper.py (hunk 1 above)
Comment (corranwebster) [inline chaco/color_mapper.py]: And looking at the comments, we may be able to just remove the whole method if it is in fact no longer used.

File: .travis.yml (hunk 2 above)
Comment (mdickinson) [inline .travis.yml]: Agreed. So long as it's no additional effort to support 3.3, I think we should do so. As soon as it becomes an effort, let's drop support.

**Comment by corranwebster** [comment]
Fixes made.  Will merge assuming tests pass cleanly unless there are objections

**Comment by mdickinson** [comment]
LGTM!

**Pull request merged**